minvws/nl-kat-coordination
# Add reports to scheduled table

**Note:** This PR is based on the code of PR #3769 

-------------------

### Changes
This PR fixes the reports that weren't showed on the Scheduled Report page.

### Issue link
Closes #3742 

### Demo
After a scheduled report has been generated, you will now see the generated reports in the scheduled table:

![afbeelding](https://github.com/user-attachments/assets/a4e11caf-2cae-416a-8f0b-2e21625e044c)


### QA notes
- Create a scheduled report
- Change the cron to * * * * *
- Wait a minute
- Go to Scheduled Report page
- Open the schedule table row
- You should see the generated report

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
